sockeye/__init__.py

```diff
@@ -11,4 +11,4 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-__version__ = '2.1.16'
```

Comment:
All changes are backwards compatible, meaning I could use a model trained with 2.1.x and it would still work with the latest code, right? If so we shouldn't bump the major version to not invalidate the already trained models of others if at all possible.

Reply:
Doesn't the change in the arguments of `MultiheadSelfAttention.hybrid_forward()` cause backwards incompatibility?

Reply:
Oh, got it. I had misunderstood what backwards compatibility meant here.